core/src/scene/drawRule.cpp

```diff
@@ -211,15 +210,17 @@ void DrawRuleMergeSet::apply(const Feature& _feature, const SceneLayer& _layer,
 
 void DrawRuleMergeSet::mergeRules(const SceneLayer& _layer) {
 
-    for (const auto& rule : _layer.rules()) {
+    size_t pos, end = m_matchedRules.size();
 
-        auto it = std::find_if(m_matchedRules.begin(), m_matchedRules.end(),
```

Comment:
What was the reason for this change?

Previously I found using `std::find_if` to be slower in comparison to doing it explicitly. Did you also discover the same?